bay/images/Dockerfile.php

```diff
@@ -36,6 +36,22 @@
     && ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
     && echo $TZ > /etc/timezone
 
-ONBUILD ARG BAY_DISABLE_FUNCTIONS
+ONBUILD ARG BAY_DISABLE_FUNCTIONS=phpinfo,pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,system,exec,shell_exec,passthru,phpinfo,show_source,highlight_file,popen,proc_open,fopen_with_path,dbmopen,dbase_open,putenv,filepro,filepro_rowcount,filepro_retrieve,posix_mkfifo
```

Comment:
Moved default values to the dockerfile, docker treats ARG declarations as `null` if not given and would define them in the environment with an empty value. This would result in `ep` adding a null value to the PHP config file.